lib/Common/Memory/HeapBlock.h

```diff
@@ -941,6 +963,32 @@ class HeapBlockList
         return tail;
     }
 
+    template <typename TBlockType>
+    static TBlockType * FindPreviousBlock(TBlockType * list, TBlockType * heapBlockToFind)
```

Comment:
Is this still used?

Reply:
nope. removed.

---
In reply to: [148163808](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148163808) [](ancestors = 148163808)